route/build/src/write_simoutput_pio.f90

```diff
@@ -203,6 +203,10 @@ SUBROUTINE output(ierr, message)
        index_write_all = arth(1,1,nRch_local)
      end if
 
+     if (trim(runMode)=='cesm-coupling') then
```

Comment:
Add a comment to just say that we are setting the timestamp of the average history output as the midpoint of the average.

Reply:
added comment: "For cesm coupling mode, the timestamp of the average history output is the midpoint of time bounds"